Profiler/CProfilerCallback.cpp

```diff
@@ -53,37 +53,42 @@ HRESULT CProfilerCallback::InitializeImplementation(IUnknown* pICorProfilerInfoU
 		return S_OK;
 	}
 
-	log.createLogFile(config);
-	log.info("looking for configuration options in: " + config.getConfigPath());
+	// Place the attach log next to the config and profiler dll
+	std::string configPath = StringUtils::removeLastPartOfPath(config.getConfigPath());
+	attachLog.createLogFile(configPath);

```

Comment:
I'm wondering whether we should write this log to the same directory as the trace log... We recently had a scenario where the profiler had to write permissions to the profiler install dir, such that logs couldn't be written there.

Reply:
According to the [issue description](https://jira.cqse.eu/browse/TS-21153), not being able to write there is fine and should just result in no attach log being written at all.
I don't think it's necessarily a good idea to "pollute" the trace folder with something else than coverage files, but that's more a feeling than a real argument.

Reply:
Hm... I'm undecided about that one. I see your point regarding the pollution. I also see me totally confused, because none of the expected files are written and I missed that the process cannot write to the install dir xD

Writing the file in many cases is still ok and better than not having it at all. I'm fine to merge this and open another issue to change the path, if such confusion happens in practice.

Reply:
Ok, then let's create a new ticket so that we can merge this as soon as possible. :)

Reply:
Let’s mit create the ticket and wait for the issue to occur in practice, first.